authors/apps/articles/filters.py

```diff
@@ -0,0 +1,27 @@
+from django_filters import FilterSet
+from django_filters import rest_framework as filters
+from .models import Article
+
+
+class ArticleFilter(FilterSet):
+    title = filters.CharFilter(lookup_expr='icontains')
+    author = filters.CharFilter('author__user__username',
```

Comment:
@mariamiah use `icontains` for look_up as well

Reply:
@mariamiah Use `icontains for all search fields

Reply:
@mariamiah Don't forget to squash your commits.